torchvision/datasets/utils.py

```diff
@@ -45,15 +47,22 @@ def download_url(url, root, filename, md5):
     if os.path.isfile(fpath) and check_integrity(fpath, md5):
         print('Using downloaded and verified file: ' + fpath)
     else:
+        pbar_ctor = tqdm_notebook if '__IPYTHON__' in locals() else tqdm
```

Comment:
@kohr-h just wonder in which situation `'__IPYTHON__' in locals()` gives True ? Test in a jupyter notebook and it gives False...

Reply:
There is an `__IPYTHON__` variable defined in an IPython console, also in notebooks. But I got it wrong, it's not in `locals()` or `globals()`, so that code didn't work.
Seems like `tqdm_notebook` is broken anyway, so I removed that bit again.